internal/converter/testdata/proto/JSONFields.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package samples;
+
+message JSONFields {
+    string name      = 1;
+    string timestamp = 2;
+    int32 id         = 3 [json_name="identifier"];
+    float some_thing = 4 [json_name="someThing"];
```

Comment:
Can you remove the `json_name` annotation?

The tool should automatically use the proper JSON representation of the field without a user having to annotate all the proto fields with the JSON name.

It might work already, but this test case does not test a case with a `snake_case` field without a `json_name` annotation.

Reply:
Alright, added a snake_case field with no JSON annotation (to test the automatic mapping).